api/src/database-models/alert.ts

```diff
@@ -10,7 +11,7 @@ export const AlertModel = z.object({
   alert_type_id: z.number(),
   name: z.string(),
   message: z.string(),
-  severity: z.enum(['info', 'success', 'error', 'warning']),
+  severity: AlertSeverity,
```

Comment:
I think this should be the `AlertSeverity` type

Reply:
updated